.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+# command to install dependencies
+install: 
+  pip install -r pip-requirements.txt
+
+# command to run tests
+script: 
+  nosetests -v --with-coverage --cover-package=presenters,plotting.figuremanager
```

Comment:
@MichaelWedel I don't mind which mechanism is followed, but I think for these MVP approaches, robustness from the get-go is so important, I think enforcing (and in fact becoming gradually stricter) on test coverage is required. Could you help @EltayebAhmed configure it in coveralls if that is the preferred approach? I've never set coveralls up myself.